PARAMETERS_MAPPING/unit_view.csv

```diff
@@ -65,3 +65,4 @@ id;name;short_name;definition;date_added
 901;Cells per cubic metre;Cells m-3;;2020-07-21
 902;Cubic micrometres;um3;;2020-07-21
 903;Picograms per litre;pg l-1;;2020-07-21
+904;Unitless;Untless;;2020-07-30
```

Comment:
If my suggestion using the `COALESCE` function in the SQL query works, then you won't need this.

Reply:
It did not work, or I did not manage to make it work. The result gives empty rows for the variables with not unit_id. The lines are there, but no information is written in them

Reply:
also, there is a typo in "Untless"